src/treq/client.py

```diff
@@ -304,19 +314,41 @@ def _convert_files(files):
         yield (param, (file_name, content_type, IBodyProducer(fobj)))
 
 
-def _combine_query_params(url, params):
-    parsed_url = urlparse(url.encode('ascii'))
-
-    qs = []
+def _coerced_query_params(params):
+    """
+    Carefully coerce *params* in the same way as `urllib.parse.urlencode()`
```

Comment:
`*params*` or ``` ``params`` ```?

Reply:
`*params*` when referring to arguments, always.